systemd/celery.service.d/main.conf

```diff
@@ -0,0 +1,7 @@
+[Service]
+Environment="User=www-data"
```

Comment:
Why not just set the [User=](https://www.freedesktop.org/software/systemd/man/systemd.exec.html#User=) option?